src/components/Results.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import moment from "moment";
+
+const Results = (props) => {
+  const res = props.results;
+  return <div>
+      <table>
+        <tr>
+          <th>title</th>
+          <th>First Name</th>
+          <th>Surname </th>
+          <th>Email </th>
+          <th>Room ID </th>
+          <th>Check in Date </th>
+          <th>Check out Date </th>
+          <th>number of days</th>
+        </tr>
+        {res.map(function (result){
```

Comment:
Think an [arrow function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions) is more idiomatic.

Reply:
A better function argument name could be `reservation` or `booking` (rather than `result`)